src/converter/flat-json.js

```diff
@@ -26,24 +26,29 @@ import { columnMajor } from '../utils';
  *    }
  * ];
  */
-function FlatJSON (arr) {
+function FlatJSON (arr, schema) {
+    if (!schema || !Array.isArray(schema)) {
```

Comment:
Here only array check is required.

Reply:
What happens if schema is an empty array?

Reply:
If schema is empty then empty data and schema `[[], []]` is returned

Reply:
Ok